src/aur.c

```diff
@@ -385,6 +404,28 @@ static int json_integer (void * ctx, long long val)
 	return 1;
 }
 
+static int json_double (void * ctx, double val)
```

Comment:
Yes, only the tags of string type should be there.
I will have to clean this function up from unneeded tags.